src/stories/Library/card-grid/card-grid.scss

```diff
@@ -0,0 +1,129 @@
+// The gap we want between the items.
+// The reason we need to define it explicity is because we need it for a
+// wrapper margin hack.
+$_card-grid-gap: $s-lg;
+
+.card-grid {
+  @include layout-container("none");
+}
+
+.card-grid__header {
+  @include layout-container();
+
+  display: flex;
+  align-items: center;
+}
+
+.card-grid__title {
+  @extend %title-underline--narrow;
+  @include typography($typo__h2);
+
+  display: flex;
+  flex-grow: 1;
+  margin-bottom: $_card-grid-gap * 2;
+}
+
+.card-grid__link,
+.card-grid__link > a {
+  @include typography($typo__links);
+  margin-bottom: $_card-grid-gap * 2;
+}
+
+.card-grid__items-wrapper {
+  // As we're using percentage values, we cannot use gap: $s-lg, as the whole
+  // flex container will expand beyond the parent.
+  // Instead, we put padding-left on each item, and then we pull the container
+  // back here, negating the first row.
+  // This is not something you can do directly on a display: flex container,
+  // so instead we have to use this ugly items-wrapper.
+  margin-left: -$_card-grid-gap;
+}
+
+.card-grid__items {
+  .card-grid--count-4 &,
+  .card-grid--count-5 &,
+  .card-grid--count-6 & {
+    justify-content: end;
+  }
+
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.card-grid__item {
```

Comment:
this is not here in this CSS class, but im jsut marking it here. 

Maybe the approach of using a border for the identity is not the best approach, since they are currently using fixed pixel values. 

For cardList i made "%identity-placeholder-pseudo", which is also using pixels, but i originally thought we could use % of the parent container to place this identity BG on images. 

Personally i think this should be done(or something similar), so small cards with no images, dont have the same huge border like the rest.

Reply:
I completely agree with you, but the design marks the borders based on PX values.
I agree with you that percentage would look better, but I think it's something we should take as part of the 'the teasers are very big' conversation on monday :)